clojure-mode.el

```diff
@@ -457,20 +457,11 @@ The command will prompt you to select one of the available sections."
 See `paredit-space-for-delimiter-predicates' for the meaning of
 ENDP and DELIM."
   (or endp
-      (not (memq delim '(?\" ?{ ?\( )))
-      (not (or (derived-mode-p 'clojure-mode)
+      (not (memq delim '(?\" ?\{ ?\()))  ;;  always insert for [
+      (not (or (derived-mode-p 'clojure-mode) ;; FIXME why does this check exist
```

Comment:
Because without it this code would be applied to any mode where `paredit` is enabled.

Reply:
Ah, I thought it was a mode local variable - will add it back in.

Reply:
Actually, wouldn't it make more sense to do `(make-local-variable 'paredit-space-for-delimiter-predicates)` inside clojure-paredit-setup rather than add global predicates?

Reply:
I guess that's an option as well.